php/php.editor/test/unit/src/org/netbeans/modules/php/editor/completion/PHPCodeCompletionTest.java

```diff
@@ -781,6 +781,7 @@ public void testClassConstructorOptionalParam_01() throws Exception {
     }
 
     public void testIssue153707_01() throws Exception {
+        // FIXME: maybe, should add only variables and $this->
         checkCompletion("testfiles/completion/lib/_base/issue153707.php", "class property: ^", false);
```

Comment:
There are unnecessary CC items. I don't think this is a big problem. I just marked it as FIXME.
#### Code
https://github.com/apache/netbeans/blob/f2611e358c181935500ea4d9d9142fb850504a72/php/php.editor/test/unit/data/testfiles/completion/lib/_base/issue153707.php#L15-L19

#### Result
https://github.com/apache/netbeans/blob/f2611e358c181935500ea4d9d9142fb850504a72/php/php.editor/test/unit/data/testfiles/completion/lib/_base/issue153707.php.testIssue153707_01.completion#L1-L218

Reply:
Agreed.